apps/web/src/pages/Users/IndexUserPage/components/UserTable.tsx

```diff
@@ -188,6 +220,7 @@ const defaultState = {
     "preferredLanguage",
     "createdDate",
     "updatedDate",
+    "rolesAndPermissions",
   ],
```

Comment:
Hiding it by default, I thought it was something people would not need to check/see that regularly